src/extension/exponent/exponentHelper.ts

```diff
@@ -63,12 +63,12 @@ export class ExponentHelper {
             .then((user) => {
                 if (!user) {
                     let username = "";
-                    return showMessage("You need to login to exponent. Please provide username and password to login. If you don't have an account we will create one for you.")
+                    return showMessage("You need to login to Expo. Please provide username and password in the input boxes after closing this window. If you don't have an account, please go to https://expo.io to create one.")
                         .then(() =>
-                            promptForInformation("Exponent username", false)
+                            promptForInformation("Expo username", false)
                         ).then((name: string) => {
                             username = name;
-                            return promptForInformation("Exponent password", true);
+                            return promptForInformation("Expo password", true);
```

Comment:
Will this prompt show the text being entered for password?

Reply:
@dhanvikapila Yes, it will.